log/sequencer_test.go

```diff
@@ -389,7 +385,6 @@ func TestIntegrateBatch(t *testing.T) {
 				merkleNodesGet:   &compactTree16,
 				updatedLeaves:    &noLeaves,
 				merkleNodesSet:   &noNodes,
-				signer:           fixedGoSigner,
```

Comment:
Is `fixedGoSigner` used anywhere else? Delete it?

Reply:
Yes, still used in `sequencer_manager_test.go`:

https://github.com/google/trillian/blob/e3eff67ad434aacbc281c39d841dcf25528d1cb6/log/sequencer_manager_test.go#L104

I thought of moving it, to keep it together, but it will be disappearing entirely soon, so I figured keeping the diff smaller was better.